ibm/mas_devops/roles/ocp_provision/README.md

```diff
@@ -23,6 +23,11 @@ Required.  Specify the version of OCP to install.  The exact format of this will
 - Environment Variable: `OCP_VERSION`
 - Default Value: None
 
+### enable_ipv6
```

Comment:
this should be under the fyre section on line 225, and made clear it is fyre only